deploy/charts/csi-driver/templates/csidriver.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: storage.k8s.io/v1beta1
+apiVersion: storage.k8s.io/v1
```

Comment:
As discussed, we should make this apiVersion over an if statement which is either v1beta1/v1 depending on the kube version being installed to. We should favor v1 at the smallest kube version we can.

https://helm.sh/docs/chart_template_guide/builtin_objects/

Reply:
Thanks Josh, I've fixed that.